src/pages/items/components/DropDown.jsx

```diff
@@ -0,0 +1,99 @@
+import { useState } from 'react';
+import useViewport from 'shared/hooks/useViewport';
+import styled from 'styled-components';
+import ArrowDownIcon from 'shared/assets/image/ic_arrow_down.svg';
+import SortIcon from 'shared/assets/image/ic_sort.svg';
+
+const DropDownButton = styled.button`
+    display: flex;
+    align-items: center;
+    gap: 24px;
+    border-radius: 12px;
+    border: 1px solid var(--gray200);
+    background-color: #fff;
+    font-size: 16px;
+    padding: ${({ $deviceType }) => {
+        if ($deviceType === 'mobile') return '0 9px';
+        if ($deviceType !== 'mobile') return '0 20px;';
+    }};
```

Comment:
삼항 연산자를 사용해주시는 것이 더 좋을 것 같습니다. 🤗

```suggestion
    padding: ${({ $deviceType }) => ($deviceType === 'mobile' ? '0 9px' : '0 20px')}
```